### ISSUE TITLE
re-install strapi and use it, web page returns error 'ReferenceError: AbortController is not defined'

### ISSUE THREAD

**Issue by Yuyang-01** [strapi/strapi#7586]
**Describe the bug**
I install strapi by quickstart guide using command 'yarn create strapi-app my-project --quickstart'.
Strapi runs well.
Then, there is a problem that i can't create a collection type, i want to uninstall strapi and reinstall.
I just rename old strapi directory and re-install strapi by quickstart guide.

When i open the page, web page returns error 'ReferenceError: AbortController is not defined'.

**Steps to reproduce the behavior**

1. Install strapi by quickstart guide using command 'yarn create strapi-app my-project --quickstart'.
2. Delete old strapi directory。
3. Re-install strapi like step 1.

**Expected behavior**
Strapi web page can run without error

**Screenshots**
![image](https://user-images.githubusercontent.com/17671854/91008600-4b470700-e611-11ea-9ab8-30a2b9d27de0.png)

**System**
- Node.js version: v12.18.2
- NPM version: 6.14.5
- Strapi version: 3.1.4
- Database:default
- Operating system:7.7.1908

**Comment by eriskhan** [comment]
What about running 'yarn install'?

**Comment by derrickmehaffy** [comment]
I am not able to reproduce this, can you please provide more information?

**Comment by Yuyang-01** [comment]
> I am not able to reproduce this, can you please provide more information?

More information like what to provide?

Yesterday i tried two ways to solve, but both failed.
1.I uninstall npm,yarn and node and delete strapi directory, then i re-install strapi by quickstart guide, the same error occurs again.
2.I change to root and add a new user, then i install strapi by quickstart guide, the same error occurs again.

**Comment by derrickmehaffy** [comment]
I'm honestly not sure in this case, I cannot reproduce it locally so I'm guessing there is some kind of problem with your node package cache? Out of curiosity are you based somewhere in Asia?

**Comment by Yuyang-01** [comment]
> I'm honestly not sure in this case, I cannot reproduce it locally so I'm guessing there is some kind of problem with your node package cache? Out of curiosity are you based somewhere in Asia?
If there is some problem with my node package cache, how to solve it? I am in Asian haha~

**Comment by Yuyang-01** [comment]
> I'm honestly not sure in this case, I cannot reproduce it locally so I'm guessing there is some kind of problem with your node package cache? Out of curiosity are you based somewhere in Asia?

If there is some problem with my node package cache, how to solve it? I am in Asian haha~

**Comment by derrickmehaffy** [comment]
I'm wondering if there is a problem with your local npm registry caching and old version. Can you check what registry your npm is using? https://docs.npmjs.com/using-npm/registry.html

**Comment by Yuyang-01** [comment]
> I'm wondering if there is a problem with your local npm registry caching and old version. Can you check what registry your npm is using? https://docs.npmjs.com/using-npm/registry.html

I run the command :nrm ls, and it shows:
* npm -------- https://registry.npmjs.org/

**Comment by derrickmehaffy** [comment]
🤔 That is quite odd, the only thing I can think of you have some cached version of packages locally. I would try cleaning out your npm cache or try using yarn as suggested above.

Make sure you don't have some very old version of strapi installed globally also.

**Comment by Yuyang-01** [comment]
> 🤔 That is quite odd, the only thing I can think of you have some cached version of packages locally. I would try cleaning out your npm cache or try using yarn as suggested above.
> 
> Make sure you don't have some very old version of strapi installed globally also.

Before this error, there is another error. When i click the 'Content-Types bulider' builder,  it returns the error as follows:

The autoReload feature is required to use this plugin. Start your server with `strapi develop`.
COLLECTION TYPES
CitiesClasses_1SCountiesCountriesDEVPLANSEnvironmentsEnvsMachinesPRODUCTSProduct_1SProjectasksReleaseNotesServerSecuritiesServersSystems

Then, i decide to delete the strapi directory and re-install it. 
Then, i meet this error.

**Comment by Yuyang-01** [comment]
> What about running 'yarn install'?

I run the comand 'yarn install' and it returns as follows:

yarn install v1.22.4
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.03s.

Then, the error still exists.

**Comment by Yuyang-01** [comment]
> 🤔 That is quite odd, the only thing I can think of you have some cached version of packages locally. I would try cleaning out your npm cache or try using yarn as suggested above.
> 
> Make sure you don't have some very old version of strapi installed globally also.

I run the command 'npm cache clean -f' and then run the command 'yarn develop &' to restart the strapi service, the error still exists.

**Comment by derrickmehaffy** [comment]
Make sure you purge the following files/directories:

- build
- .cache
- node_modules
- package-lock.json
- yarn.lock

Then rerun `yarn` , `yarn build`, `yarn develop`

**Comment by Yuyang-01** [comment]
> yarn build

I try purge the above 5 files/directories, the error still exists, i wonder if there is something wrong with AbortController?
The error returns 'ReferenceError: AbortController is not defined'

**Comment by Yuyang-01** [comment]
> Make sure you purge the following files/directories:
> 
> * build
> * .cache
> * node_modules
> * package-lock.json
> * yarn.lock
> 
> Then rerun `yarn` , `yarn build`, `yarn develop`

I try purge the above 5 files/directories, the error still exists, i wonder if there is something wrong with AbortController?
The error returns 'ReferenceError: AbortController is not defined'

**Comment by derrickmehaffy** [comment]
@Yuyang-01 not that I can see, I've tried every possible combination to try and reproduce the issue and I can't. This has to be something environment specific (or requires a very specific set of steps to reproduce)

**Comment by augustinnoha** [comment]
Hi all, discovering this amazing tool and I have the same problem (using NPX). This problem occures on my business laptop but working perfectly on my personal Mac.

Environment on my pro laptop :
Using Laragon with
- node v12.5.0
- npm v6.9.0
- windows NT w32

Guess it is platform related but your help or ideas are welcom

**Comment by son8292003** [comment]
Hi all,

I building my new project with this nice tool and I encountered the same issue. 

I found a post on stack overflow [https://stackoverflow.com/questions/55718778/why-abortcontroller-is-not-defined](https://stackoverflow.com/questions/55718778/why-abortcontroller-is-not-defined) about AbortController. The second post stated: 

"The MDN documentation on AbortController includes an up-to-date table of supported browsers.

The API is still marked as experimental, although it should be well supported in current browsers. Firefox has it since November 2017 (FF 57), and Chrome followed on April 2018 (Chrome 66)."

My OS is ubuntu 16.04 and my chrome version was 63. After updating my chrome to version 85, my issue is gone. I believe this issue is caused the browser, not strapi.

@Yuyang-01 @augustinnoha  What browser and version are you using? I would suggest testing the issue again with the latest version of chrome.
